### PR TITLE
Train 188

### PR DESCRIPTION
**Pull request by orsenthil**
## High Level Description

* Merge #1572 - Bumps dcos-installer-ui package 
* Merge #1717 - Bump dcos-metrics version
* Merge #1707 - SetTruncateBit: true in mesos-dns
* Merge #1722 - Include DCOS-UI/Marathon team members to owners.json file 
* Merge #1709 - DCOS_OSS-1246: Update DC/OS UI Package

### CODE REVIEW

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Comment by orsenthil** [comment]
@mesosphere-mergebot label ready for review

File: owners.json [hunk 1]
```diff
@@ -3,7 +3,10 @@
   "adam-mesos": {"slack": "adam"},
   "branden": {"slack": "branden"},
   "lingmann": {"slack": "jeremy"},
+  "MatApple": {"slack": "mat"},
   "mellenburg": {"slack": "mellenburg"},
+  "meichstedt": {"slack": "matthias.eichstedt"},
+  "orlandohohmeier": {"slack": "orlando"},
```
Comment (jgehrcke) [inline owners.json]: :-)!!

**Comment by orsenthil** [comment]
@lingmann / @BenWhitehead  - this is ready for review.

**Comment by spahl** [comment]
@mesosphere-mergebot bump-ee

**Pull request merged**